mod-orders-storage/schemas/piece.json

```diff
@@ -54,6 +54,18 @@
       "type": "string",
       "description": "Chronology is the descriptive information for the dating scheme of a serial. Synchronized with inventory item."
     },
+    "barcode": {
+      "type": "string",
+      "description": "Barcode is the descriptive information for the barcode of a serial. Synchronized with inventory item."
+    },
+    "accessionNumber": {
+      "type": "string",
+      "description": "AccessionNumber is the descriptive information for the accession number of a serial. Synchronized with inventory item."
+    },
+    "callNumber": {
```

Comment:
In Item schema this field is named as: itemLevelCallNumber, maybe @usavkov-epam knows the correct name